app/conf/Configuration.scala

```diff
@@ -47,8 +47,13 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
   }
 
   object environment {
-    val stage = stageFromProperties.toLowerCase
+    val stage = stageFromProperties
```

Comment:
This is actually a bit annoyingly complicated because we don't want the `stage` value to distinguish between code and dev environments. This is because `stage` helps us distinguish if we are reading and writing from the code bucket (on code and dev) or the prod bucket (prod only). I've made a slightly ugly fix for this - let me know what you think...

Reply:
LGTM